### PR TITLE
Docs: constrain homepage content width on larger screens

### LINKED ISSUE #923: Docs - Homepage content too wide on ultrawide screens
### Before creating a new issue, please confirm:

- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-ui/issues?q=is%3Aissue+) and [discussions](https://github.com/aws-amplify/amplify-ui/discussions).
- [X] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md#bug-reports).

### On which framework/platform are you having an issue?

Other

### Which UI component?

Other

### Please describe your bug.

Content feels too wide on Ultrawide monitors (using a 3440px ultra hd screen).

### What's the expected behaviour?

We should put a max-width on the docs homepage content

### Help us reproduce the bug!

Visit the Homepage of the docs

### Code Snippet

_No response_

### Additional information and screenshots

Screen Shot 2021-12-01 at 2 32 14 PM

### PR DESCRIPTION
**Pull request by reesscot**
*Issue #, if available:*
Closes #923 

*Description of changes:*
This PR constrains the width of the homepage content to prevent it from getting too wide on large screens.

*Also fixed:*
* Horizontal scrollbar bug caused by vertical scrollbar (`100vw` => `100%`)
* Made "Looking for other Amplify Products?" list collapse to single column on smaller screens

https://user-images.githubusercontent.com/6165315/144944106-e983f74d-7d76-4b97-be66-b7dbe873d9c6.mp4

https://user-images.githubusercontent.com/6165315/144944118-0673121c-bb31-4daa-8ad5-6d7620f33d00.mp4

https://user-images.githubusercontent.com/6165315/144944094-512d5f7a-5838-47e9-9a2f-a0ae6d0b039f.mp4

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/src/styles/docs/base.scss [hunk 1]
```diff
@@ -196,7 +193,13 @@ h3:hover > a .icon-link {
 
   --amplify-components-heading-1-font-size: 3rem;
   --amplify-components-heading-1-font-weight: 750;
-  
+
   /* custom vars */
   --docs-header-height: 4rem;
 }
+
+.container {
```
Comment (reesscot) [inline docs/src/styles/docs/base.scss:201]: This is the only non-prettier change in this file.

File: docs/src/styles/docs/header.scss [hunk 2]
```diff
@@ -2,15 +2,15 @@
   display: flex;
   position: sticky;
   top: 0;
-  width: 100vw;
+  width: 100%;
```
Comment (reesscot) [inline docs/src/styles/docs/header.scss:5]: This fixes a horizontal scrollbar bug. 100vw doesn't subtract for scollbars, so when a vertical scrollbar is present it's too wide and the page shows a dreaded horizontal scrollbar. In this case setting this to 100% of the parent container works great.

File: docs/src/pages/index.page.tsx [hunk 3]
```diff
@@ -199,34 +199,36 @@ const HomePage = ({ colorMode, setThemeOverride, themeOverride }) => {
...
-            customSetup={{
-              dependencies: {
-                '@aws-amplify/ui-react': 'next',
-                'aws-amplify': 'latest',
-              },
-              entry: '/index.js',
-            }}
-          />
-        </Card>
+        <View className="container">
```
Comment (reesscot) [inline docs/src/pages/index.page.tsx:202]: The only think that changed here was adding this `View` around the `Card`

File: docs/src/styles/docs/base.scss (hunk 1 above)
Comment (dbanksdesign) [reply docs/src/styles/docs/base.scss:201]: Dang did prettier not run on my commit to this file?

File: docs/src/styles/docs/header.scss (hunk 2 above)
Comment (dbanksdesign) [reply docs/src/styles/docs/header.scss:5]: 🙌

**Review by dbanksdesign — APPROVED** [review]
LGTM! 

Glad someone is fixing my sloppy code 😝

**Pull request merged**